docs/advanced.rst

```diff
@@ -650,3 +650,12 @@ The ``-s`` is necessary because the CherryPy class also wraps stdin and stdout.
    If you want to really unit test your CherryPy application, meaning without
    having to start a server, you may want to have a look at
    this `recipe <https://bitbucket.org/Lawouach/cherrypy-recipes/src/tip/testing/unit/serverless/>`_.
+
+.. note::
+
+   The :class:`helper class <cherrypy.test.helper.CPWebCase>` derives from 
+   ``unittest.TestCase`` class. For this reason, running from  `py.test <http://pytest.org/latest/>`_,
+   there are some limitations with respect to standard  ``pytest`` tests, especially if you are grouping
+   the tests in test classes. 
+   You can find more details at `this page <https://docs.pytest.org/en/latest/unittest.html#pytest-features-in-unittest-testcase-subclasses>`_.
```

Comment:
I believe it's possible to multi-line it like this:
```suggestion
   You can find more details at `this page 
   <https://docs.pytest.org/en/latest/unittest.html#pytest-features-in-unittest-testcase-subclasses>`_.
```
Also, using intersphinx should allow us to have shorter refs: https://github.com/cherrypy/cherrypy/blob/db0a8a6/docs/conf.py#L101-L104